packages/algo-ts/src/impl/state.ts

```diff
@@ -60,12 +69,13 @@ export class LocalStateCls<ValueType> {
 }
 
 export class LocalStateMapCls<ValueType> {
-  #value = new Map<bytes, LocalStateCls<ValueType>>()
+  #value = new Map<string, LocalStateCls<ValueType>>()
 
   getValue(account: Account): LocalStateCls<ValueType> {
-    if (!this.#value.has(account.bytes)) {
-      this.#value.set(account.bytes, new LocalStateCls<ValueType>())
+    const accountString = uint8ArrayToHex(BytesCls.fromCompat(account.bytes).asUint8Array())
```

Comment:
We can use BytesCls.valueOf() here

Reply:
Presumably `account.bytes.valueOf()` would work actually

Reply:
Nice. I will update the code in the testing repo which does the same thing.